packages/e2e-test-utils-playwright/src/editor/update-post.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Publishes the post, resolving once the request is complete (once a notice
+ * is displayed).
+ *
+ * @param this
+ */
+export async function updatePost( this: Editor ) {
+	const updateButton = this.page.getByRole( 'button', {
+		name: 'Update',
+	} );
+	await updateButton.click();
+
+	await this.page
+		.getByRole( 'button', {
+			name: 'Update',
+		} )
+		.waitFor();
```

Comment:
I would recommend matching the helper from block binding e2e tests. Why?

* Uses scoped selectors.
* Waiting for notices turned out to be less flaky than waiting for label changes.

https://github.com/WordPress/gutenberg/blob/11f3096a6cb8f01afbd15ec4a440dade45ef1730/test/e2e/specs/editor/various/block-bindings.spec.js#L1933-L1942

Reply:
It would probably be better to click (close) the dismiss notice too, so that we don't end up with duplicating ones. 👍

Reply:
Done 👍